frontend/pages/SoftwarePage/components/AddPackageAdvancedOptions/AddPackageAdvancedOptions.tsx

```diff
@@ -16,8 +16,8 @@ import { IAddPackageFormData } from "../AddPackageForm/AddPackageForm";
 
 const getSupportedScriptTypeText = (pkgType: PackageType) => {
   return `Currently, ${
-    isWindowsPackageType(pkgType) ? "Power" : ""
-  }Shell scripts are supported.`;
+    isWindowsPackageType(pkgType) ? "PowerS" : "s"
+  }hell scripts are supported.`;
```

Comment:
```suggestion
    isWindowsPackageType(pkgType) ? "PowerShell" : "shell"
  } scripts are supported.`;
```

I appreciate the character efficiency, but I think it would be a little easier to read if slightly more verbose.